### PR TITLE
Replace Old Instructions

### PR DESCRIPTION
**Pull request by KevinSirius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,43 +78,49 @@ docker-compose up
 
 It's going to take 1-2 minutes until everything is initialized, this includes the postgres initialization, the caches
 announcing themselves to the bootstrap service, the publisher requesting a list of all caches from the bootstrap service
-and finally the publisher and the caches negotiating an escrow, and a distributed tracing backend store. After the escrow
-is setup you can download from the caches using `cachecash-curl` or [typescript-cachecash]:
```
Comment (JustinCappos) [inline README.md:58]: Why was this content cut?  Can't you still do these steps to test?

File: README.md [hunk 2]
```diff
@@ -54,43 +78,49 @@ docker-compose up
...
-You will also need some extra code generation tools:
+Run:
 
 ```bash
-make dev-setup
+make cachecash-curl
 ```
 
-To generate source from proto files:
+## Test Network Locally
```
Comment (JustinCappos) [inline README.md]:
```suggestion
## Test CacheCash Content Retrieval Locally
```

File: docker-compose.yml [hunk 3]
```diff
@@ -31,6 +31,7 @@ services:
     image: postgres:11
     environment:
     - POSTGRES_DB=kvstore
+    - POSTGRES_HOST_AUTH_METHOD=trust
```
Comment (JustinCappos) [inline docker-compose.yml:34]: This eventually will have security ramifications if left in.  I feel like we need more documentation or perhaps a different workflow around this...

File: README.md (hunk 1 above)
Comment (KevinSirius) [reply README.md:58]: Yes. Sorry this edit was done before yesterday when I made cachecash-curl work. I will change this.

File: docker-compose.yml (hunk 3 above)
Comment (KevinSirius) [reply docker-compose.yml:34]: Ok I will write more about the postgres in the readme. For now when we show this repo to other people helping on the project, I think as long as this work for local testing we should be fine..? Maybe when we decide we can successfully deploy this software we can update the workflow.

**Comment by lukpueh** [comment]
Hi @KevinSirius, I'm also in the process of setting up a local CacheCash test network. Any updates on the instructions?

**Comment by KevinSirius** [comment]
> Hi @KevinSirius, I'm also in the process of setting up a local CacheCash test network. Any updates on the instructions?

Hi. Please check my fork or the latest PR for reference.

**Pull request closed without merging**